packages/uniforms/__tests__/BaseForm.tsx

```diff
@@ -31,11 +31,11 @@ describe('BaseForm', () => {
 
   it('have correct context', () => {
     const wrapper = mount<BaseForm<any>>(
-      <BaseForm error={error} model={model} schema={schema} />,
+      <BaseForm error={error} model={model} schema={schema} />
     );
 
     const context = wrapper.instance().getContext();
-    expect(context).toEqual({
+    expect(context).toEqual<Context<any>>({
```

Comment:
this way, the expected object is type checked. This helps to not forget properties